src/EnergyPlus/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ if(WIN32)
 endif()
 
 set(PYTHON_API_VERSION_MAJOR 0)
-set(PYTHON_API_VERSION_MINOR 1)
+set(PYTHON_API_VERSION_MINOR 2)
```

Comment:
Since the API changed, I incremented this.  In the very near future we can be more **semantic** about these things.